readers/influxdb/README.md

```diff
@@ -76,6 +91,11 @@ docker-compose -f docker/docker-compose.yml up -d
 docker-compose -f docker/addons/influxdb-reader/docker-compose.yml up -d
 ```
 
+And, to use the default .env file, execute the following command:
+```bash
+docker-compose -f docker/addons/influxdb-reader/docker-compose.yml up --env-file docker/.env -d
+```
+
```

Comment:
Is this `[doc]` link at the end needed, looks like we don't use it?

Reply:
This is just a link, it's not used anywhere in the file (there is no [text](doc) pattern). It does make sense to keep it, but we need to add a ref like this:
```
Official docs can be found [here](doc).
```